Command/ResourcesListCommand.php

```diff
@@ -78,7 +86,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $files = $this->retrieveFiles($dirs);
 
         foreach ($files as $file) {
-            $path = str_replace($basePath, '%kernel.root_dir%', $file);
+            $path = str_replace($basePath, '%kernel.project_dir%', $file);
```

Comment:
I guess this wasn't working before because it was replacing `$basePath` which contained `'%kernel.root_dir%/..'` with `'%kernel.root_dir%'`